.github/workflows/test.yaml

```diff
@@ -4,17 +4,9 @@ on:
   workflow_dispatch:
     inputs:
       version:
-        description: "Radius version number to use (e.g. 0.1.0, 0.1.0-rc1, edge). Defaults to edge."
+        description: "Radius version number to use (e.g. 0.1.0, 0.1.0-rc1, edge)."
         required: false
-        default: "edge"
         type: string
-  push:
```

Comment:
I don't think we need to run on push

Reply:
I still think we should test on pushes. even if the PR runs past, we've sometimes seen failures on pushes in other repos and runs

Reply:
ok, I re-added this trigger